ci/kind/kind-setup.sh

```diff
@@ -300,6 +300,19 @@ function configure_vlan_subnets {
       docker_run_with_host_net iptables -t filter -A FORWARD -i ${vlan_interfaces[j]} -o ${vlan_interfaces[i]} -j ACCEPT
     done
   done
+
+  # With FlexibleIPAM, Antrea SNAT is disabled (noSNAT: true) so Pods don't have access to the external network by default (including regular / NodeIPAM Pods). 
+  # With our configuration, Docker will already perform SNAT for FlexibleIPAM Pods. However, we need a custom SNAT rule for regular Pods.
```

Comment:
I think you misunderstood, this comment is specifically for the _second_ iptables rule below.

For the first iptables rule, maybe a comment like this one:
> Bypass default Docker SNAT rule for FlexibleIPAM traffic from the untagged subnet (192.168.240.0/24, which is the subnet for the Docker bridge network) and destined to the VLAN subnets (192.168.241.0/24, 192.168.242.0/24).

Reply:
added